x/stake/handler.go

```diff
@@ -150,6 +148,7 @@ func handleMsgEditValidator(ctx sdk.Context, msg types.MsgEditValidator, k keepe
 			return err.Result()
 		}
 		validator.Commission = commission
+		k.OnValidatorModified(ctx, msg.ValidatorAddr)
```

Comment:
probably shouldn't be calling this before k.SetValidator().  Also hooks being called from handlers is weird.

Reply:
I think this is the correct order, but we should change the name of the hook - https://github.com/cosmos/cosmos-sdk/issues/2538.